third_party/dependency_analyzer/src/test/io/bazel/rulesscala/dependencyanalyzer/AstUsedJarFinderTest.scala

```diff
@@ -293,6 +293,14 @@ class AstUsedJarFinderTest extends FunSuite {
     )
   }
 
+  test("static annotation of inherited class is indirect") {
+    checkIndirectDependencyDetected(
+      aCode = "class A extends scala.annotation.StaticAnnotation",
```

Comment:
Also,
This test failed originally? If so which dep did it complain about? Doesn’t StaticAnnotation come from std lib? Feel free to correct me if I’m completely off

Reply:
The test failed originally, as we expect it to complain that A is an unused dependency, but it did not. And A should be an unused dependency because if you look at C's code it never touches A.